### DISCUSSION TITLE
두번째 미팅 후기

### DISCUSSION THREAD

**Discussion by jongfeel** [humansof42seoul/Humansof42-v2#15] [General]
사실 오픈 프로젝트 얘기는 1시간 가량 했지만 저는 그보다 같이 저녁 하면서 들었던 얘기들이 더 값진 얘기들이라고 생각하고 있습니다.

또 여러분들이 그래도 어느 정도 남들에게 사회 경험이라는 걸 해봤다? 라는 걸 얘기할 수 있는 수준에서 모인 팀이라 많은 얘기를 들을 수 있었던 것 같습니다

여러모로 재밌었고 애정이 많이 가는 팀인거 같아요

다음에는 돼지고기를 먹으면서 또 얘기할 수 있는 기회가 있으면 좋겠습니다

**Comment by jongfeel** [comment]
아 그리고 @jwon42 님 바지 주신거 감사합니다. 생각지도 못했는데 주셔서 너무 좋았어요

**Comment by jwon42** [comment]
멘토님께서 좋은 말씀과 조언을 많이 해주셔서 프로젝트를 초월한 멘토링을 받고 있는 것 같아요. 정말 감사드립니다. 🙏
